bin/p2-dsctl/main.go

```diff
@@ -55,11 +59,19 @@ var (
 
 	cmdUpdate       = kingpin.Command(CmdUpdate, "Update a daemon set.")
 	updateID        = cmdUpdate.Arg("id", "The uuid for the daemon set").Required().String()
-	updateAZ        = cmdUpdate.Flag("az", "The availability zone of the pod cluster").String()
+	updateSelector  = cmdUpdate.Flag("selector", "The node selector, uses the same syntax as the test-selector command").Default(SelectorNotSpecified).String()
```

Comment:
Empty selectors are valid in the case where you want a daemon to go to every node in the label tree, and I suspect there is a need to detect when the flag was passed or not. Sorry about the kingpin confusion @KoishiKomeiji